server/models/TestIssue.js

```diff
@@ -0,0 +1,41 @@
+module.exports = function(sequelize, { BOOLEAN, INTEGER }) {
+    return sequelize.define(
+        'TestIssue',
+        {
+            id: {
+                type: INTEGER,
+                allowNull: false,
+                primaryKey: true,
+                autoIncrement: true
+            },
+            test_id: {
+                type: INTEGER,
+                allowNull: false,
+                references: {
+                    model: 'test',
+                    key: 'id'
+                }
+            },
+            run_id: {
+                type: INTEGER,
+                allowNull: false,
+                references: {
+                    model: 'run',
+                    key: 'id'
+                }
+            },
+            issue_number: {
+                type: INTEGER,
+                allowNull: false
+            },
+            open: {
+                type: BOOLEAN,
+                allowNull: true
```

Comment:
I believe boolean false is *not* null, so `allowNull` can be false for consistency / safety.